README.md

```diff
@@ -281,7 +281,7 @@ UDATA_URI_API_KEY = "example.api.key"
 SENTRY_DSN = "https://{my-sentry-dsn}"
 ```
 
-The webhook integration sends HTTP messages to `udata` when resources are analyzed or checked to fill resources extras.
+The webhook integration sends HTTP messages to `udata` when resources are analysed or checked to fill resources extras.
```

Comment:
American english or queen's english, i won't take sides 😬 (ok, i'd rather use the queen's english, but that's just imho 🤣 )

Reply:
Ahah I hesitated, but since previous code was using mostly British English, I decided to go more consistent and follow British English...